src/main/java/org/apache/accumulo/testing/randomwalk/multitable/DropTable.java

```diff
@@ -34,7 +34,7 @@ public void visit(State state, RandWalkEnv env, Properties props) throws Excepti
     List<String> tables = (List<String>) state.get("tableList");
 
     // don't drop a table if we only have one table or less
```

Comment:
This comment might need to be updated to reflect the new  if condition.

Reply:
Fixed in bf9f3fc